dxr/build.py

```diff
@@ -360,8 +355,8 @@ def _unignored_folders(folders, source_path, ignore_filenames, ignore_paths):
 
 
 def file_contents(path, encoding_guess):  # TODO: Make accessible to TreeToIndex.post_build.
-    """Return the unicode contents of a file if we can figure out a decoding.
-    Otherwise, return the contents as a string.
+    """Return the unicode contents of a file if we can figure out a decoding,
```

Comment:
One last thing: since we changed the semantics, we should rename this `unicode_contents()` to reflect them. That also provides contrast with `file_contents_at_rev()`, which might still be on the binary-returning bandwagon.